director/origin_api_test.go

```diff
@@ -128,8 +128,13 @@ func TestCreateAdvertiseToken(t *testing.T) {
 
 	// Generate a private key
 	viper.Set("IssuerKey", kfile)
-	err := config.GeneratePrivateKey(kfile, elliptic.P521())
-	assert.NoError(t, err)
+	config.InitConfig()
+	err := config.InitServer(ctx, config.DirectorType)
+	require.NoError(t, err)
+
+	// Launcher will set default values to some of the server urls. Reset here.
+	viper.Set("Federation.RegistryUrl", "")
+	viper.Set("Federation.DirectorURL", "")
 
 	// Test without a namsepace set and check to see if it returns the expected error
 	tok, err := CreateAdvertiseToken("test-namespace")
```

Comment:
I guess this isn't in the purview of this particular PR, but this is an invalid namespace because it doesn't have a preceding `/`. Are there any consequences of testing things at this level of code with invalid namespaces? I guess the token doesn't really care whether the namespace is valid?